src/Models/Constants/Constants.ts

```diff
@@ -258,12 +258,16 @@ export const ViewerModeBackgroundColors: Array<IADTBackgroundColor> = [
     }
 ];
 
-export const EnvironmentsLocalStorageKey = 'cb-environments'; // TODO: These will be deprecated with the new local storage structure
-export const ContainersLocalStorageKey = 'cb-containers'; // TODO: These will be deprecated with the new local storage structure
-export const StorageAccountsLocalStorageKey = 'cb-storage-accounts'; // TODO: These will be deprecated with the new local storage structure
-
-export const SelectedEnvironmentLocalStorageKey = 'cb-selected-environment'; // TODO: These will be deprecated with the new local storage structure
-export const SelectedContainerLocalStorageKey = 'cb-selected-container'; // TODO: These will be deprecated with the new local storage structure
+export const EnvironmentsLocalStorageKey =
```

Comment:
can we put the comments above each item instead of inline? I don't think this is going to link the comments to the right items.